x-pack/plugins/observability_solution/apm/server/routes/span_links/get_linked_parents.ts

```diff
@@ -58,8 +56,7 @@ export async function getLinkedParentsOfSpan({
     },
   });
 
-  const fields = response.hits.hits[0]?.fields;
-  const fieldsNorm = normalizeFields(fields) as unknown as TransactionRaw | SpanRaw | undefined;
+  const fieldsNorm = linkedParentsOfSpanMapping(response.hits.hits[0]?.fields);
```

Comment:
@AlexanderWert I remember we discussed that we don't support the span links ATM so should we find a way to "skip" them or "hide" the tab in case of otel data (Together with @miloszmarcinkowski we found another UI Error we have here because of SpanLink so I am wondering how to handle that in this case)

Reply:
yes, let's just hide stuff related to Links in the UI and NOT change the query logic for SpanLinks for now